srcpkgs/vscode/patches/ripgrep.patch

```diff
@@ -1,31 +1,25 @@
-Ping atk if this needs updating.
-
-This prevents vscode from fetching prebuilt ripgrep from Microsoft
-during build, which unbreaks build on platforms where MS deos not
-ship a prebuilt ripgrep.
-
```

Comment:
Why remove these comments?

Reply:
Because he removed the actual solution so that vscode uses its own version of ripgrep again. Not sure if I concur; we usually prefer to use our own version.

Reply:
Sorry removed it by mistake.. Gonna readd it again.

@atk I still use your version.

Reply:
Thanks, if it's up to me, I approve.